src/pages/auth/Login.js

```diff
@@ -3,43 +3,32 @@ import { Trans, t } from '@lingui/macro';
 import { noop } from 'utils/noop';
 import { I18n } from '@lingui/react';
 
+import { SignUp, TitleMessage } from './Utilities';
 import Button from 'components/button/Button';
 import Facebook from 'components/icon/Facebook';
 import Instagram from 'components/icon/Instagram';
 import TextField from 'components/textfield/TextField';
 import Twitter from 'components/icon/Twitter';
 import { Panel, PanelContent } from 'components/panel/Panel';
 
-import { ReactComponent as Logo } from '../home/assets/images/coding-coach-logo.svg';
-
 export default function Login() {
   return (
     <div className="h-screen bg-gradient-primary py-3">
       <div className="container">
-        <div className="flex justify-between mb-12">
-          <Logo className="w-12 h-8" />
-          <Button onClick={noop} shape="pill" size="small" typography="none">
-            <Trans id="auth.signup.title" defaults="Sign Up" />
-          </Button>
-        </div>
+        <SignUp />
         <div className="md:flex md:flex-col md:items-center">
+          <TitleMessage
+            id="auth.signin.description"
+            defaults="Sign in or create a new account to get started."
+          />
           <div className="px-2 md:w-3/5 lg:w-1/3">
-            <h1 className="text-white font-semibold mb-2">
-              <Trans id="site.title" defaults="Coding Coach" />
-            </h1>
-            <p className="text-white font-sm w-3/4 mb-8">
-              <Trans
-                id="auth.signin.description"
-                defaults="Sign in or create a new account to get started."
-              />
-            </p>
             <Panel floating>
               <PanelContent>
                 <div className="flex mb-6">
                   <h2 className="flex-1 text-2xl font-semibold text-primary">
                     <Trans id="auth.signin.title" defaults="Sign In" />
                   </h2>
-                  <a href="#noop" className="text-sm h-4 mt-1">
+                  <a href="recovery" className="text-sm h-4 mt-1">
```

Comment:
Please use the `Link` component from `reach-router`.

Reply:
Stupid question but, which is the benefit of it? In my side projects I use the `<a>` too! Could you light up a little bit please @crysfel ?

Reply:
If using an `a` element, there will be a full reload of the next page (Which is not ideal). If using `Link` it will only render the parts that are changing.

Reply:
basically the `<a>` will execute a window.location or similar... but I thought that `Link` would do the same... so does it keep references from nodes and things like this? @crysfel

Reply:
You would use `<a>` when linking to a different website. And `Link` for internal links, because it doesn't reload the react app, it works with the `History` api: https://css-tricks.com/using-the-html5-history-api/#article-header-id-1

Reply:
Tomorrow I'll check this. Thank you so much @crysfel  :smile:

Reply:
Good information @crysfel . I believe since Link is part of React, it has the intelligence to re render only the parts that changed. Good to know.